tests/js/spec/components/deprecatedforms/projectMapperField.spec.jsx

```diff
@@ -72,5 +72,22 @@ describe('ProjectMapperField', () => {
     expect(onChange).toHaveBeenCalledWith([['24', '1']], []);
   });
 
-  it('handles deleted items without error', () => {});
+  it('allows a single Sentry project to map to multiple items but not the value', () => {
+    existingValues = [['24', '1']];
+    wrapper = mountWithTheme(<RenderField {...props} value={existingValues} />);
+    // can find the same project again
+    openMenu(wrapper, {control: true, name: 'project'});
+    expect(
+      findOption(wrapper, {value: '24'}, {control: true, name: 'project'})
+    ).toHaveLength(1);
+    // but not the value
+    openMenu(wrapper, {control: true, name: 'mappedDropdown'});
+    expect(
+      findOption(wrapper, {value: '1'}, {control: true, name: 'mappedDropdown'})
+    ).toHaveLength(0);
+    // validate we can still find 2
+    expect(
+      findOption(wrapper, {value: '2'}, {control: true, name: 'mappedDropdown'})
+    ).toHaveLength(1);
+  });
```

Comment:
In this test I'm a little confused what 24, 1, and 2 are. 24 is 'project' and 1 and 2 are both 'mappedDropdown' - is 24 the Sentry project and 1 and 2 are the Vercel projects, or the other way around?

Reply:
@ceorourke 23/24 are the Sentry projects and 1/2 are the Vercel projects (aka the mapped value)